lib/components/result-view/display.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import React from "react";
+import {
+  DisplayData,
+  ExecuteResult,
+  StreamText,
+  KernelOutputError,
+  Output
+} from "@nteract/outputs";
+import { MediaComponents } from "./transforms";
+
+type Props = {};
+
+class Display extends React.PureComponent<Props> {
+  render() {
+    return (
+      <Output {...this.props}>
+        <ExecuteResult expanded>{MediaComponents()}</ExecuteResult>
+        <DisplayData expanded>{MediaComponents()}</DisplayData>
+        <StreamText expanded />
+        <KernelOutputError expanded />
+      </Output>
```

Comment:
Check out [this example](https://github.com/nteract/nteract/blob/a44db1c20aa6d30c78711958a55f0ddf6fff33cc/packages/outputs/__tests__/output.spec.tsx#L112-L128). 

We would support more output media types, what used to be called "transforms". Below we are listing explicitly each media type we support, and `RichMedia` will automatically select the "richest" one based on the order they are declared below:

```jsx
<RichMedia>
    <Vega3/>
    <Vega2/>
        ...
    <Media.Json />
    <Media.HTML />
    <Media.Plain />
</RichMedia>
```

This may eliminate the need for the `transforms.js` file. Unless the list of components would need to be reused outside this function.

Reply:
@BenRussert I feel like the `transforms.js` simplifies each component by calling that function. This function is also used in another file too. However, I also keep a `MediaArray` in case we want easier access to their properties. Instead, I would rather name the file to `media-types.js` since transforms is no longer the correct naming convention. 

Also I don't understand if this was meant for me. 
>Check out this example.
>
>We would support more output media types, what used to be called "transforms". Below we are listing explicitly each media type we support, and RichMedia will automatically select the "richest" one based on the order they are declared below:

I just don't know what you mean here since where you provided your example Output is required and ExecuteResult and DisplayData are wrappers for RichMedia